luarules/gadgets/team_transfer/mod_resource_tax.lua

```diff
@@ -58,34 +58,6 @@ local function ValidateResourceTax(oldTeam, newTeam, resourceType, amount)
     return true -- Always allow, just log tax info
 end
 
--- Command restrictions related to tax policy
```

Comment:
Let's actually folow the factoring here: https://gist.github.com/keithharvey/ab12d0f3113715dd1d0abd1717792a9b and put this in the correct mod_option. I kind of just want to see that functional list as part of this PR